quetz/main.py

```diff
@@ -1839,4 +1839,14 @@ def serve_channel_index(
     return serve_path("index.html", channel, accept_encoding, session, dao)
 
 
+@app.get("/health/ready", status_code=status.HTTP_200_OK)
+def is_ready(task: Task = Depends(get_tasks_worker)):
+    return {}
```

Comment:
I'm unsure about the correct definition and readiness and liveness for Quetz. [But it seems like when a task worker can be instantiated](https://github.com/mamba-org/quetz/blob/87ec595410ba39de246bdda221f58735cb3981a0/quetz/deps.py#L84-L91), the whole service should be available.